source/code/plugins/in_npmd_server.rb

```diff
@@ -452,7 +468,8 @@ def process_dsc_command(cmd)
                 _ind = _req.index(":")
                 _msg = _req[_ind+1..-1]
                 unless _ind.nil? or _ind + 1 >= _req.length
-                    log_error "dsc:#{_msg}"
+                    Logger::logInfo "dsc:#{_msg}"
```

Comment:
Can you explain why you're changing this to an info-level message instead of an error-level message?